### PR TITLE
Fixed dapl tests script and parameters

### PR DESCRIPTION
**Pull request by manvanthar**
Fixed & optimised dapl script code.  added additional scenarios to dapl yaml files.

Signed-off-by: Manvanthara Puttashankar <manvanth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by manvanthar** [comment]
JOB ID : 3c31913468a334ca07b371a8842f2e672b29c324
JOB LOG : /root/avocado-fvt-wrapper/results/job-2019-07-16T08.39-3c31913/job.log
(1/5) dapl.sh;first-73b9: PASS (11.46 s)
(2/5) dapl.sh;second-3467: PASS (12.49 s)
(3/5) dapl.sh;third-0e32: PASS (22.26 s)
(4/5) dapl.sh;fourth-c93b: PASS (11.48 s)
(5/5) dapl.sh;fifth-bd3f: PASS (3.14 s)
RESULTS : PASS 5 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME : 61.67 s
JOB HTML : /root/avocado-fvt-wrapper/results/job-2019-07-16T08.39-3c31913/results.html

**Comment by manvanthar** [comment]
[job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/3401432/job.log)

Job log attached

File: io/net/infiniband/dapl.sh [hunk 1]
```diff
@@ -70,12 +55,13 @@ dapl_exec()
...
-        if type "dapltest" > /dev/null
+        if type "dapltest" > /dev/null || [[ `python -c 'from avocado.utils.software_manager import SoftwareManager; \
+               print SoftwareManager().install("dapl*")'` == 'True' ]]
         then
-            dapl_exec dapltest "$host_param" "$peer_param"
+            dapl_exec dapltest "$peer_param" "$host_param"
         else
-            avocado_debug "Cmd dapltest doesn't exist, Test will be skipped"
-            exit 1
+           avocado_debug 'Dapl Packages could not be installed, Test will be skipped'
```
Comment (narasimhan-v) [inline io/net/infiniband/dapl.sh]: @manvanthar can you take care of indentation here ?

**Comment by manvanthar** [comment]
@narasimhan-v changes made to new pull request PR #1213

**Pull request closed without merging**